### ISSUE TITLE
Contact aliases

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#108]
```
Imagine the scenario:

You have three contacts named "Susan Boyle", "Susan Jameson", "Susan Jones".

Now, in order to SMS these, you have to type, for instance: sms:Susan 
Boyle:Hello there

Now, imagine I'm not interested in SMSing any of the Susans, apart from Boyle; 
then it would be convenient to have "sms:Susan:Hello there" go to Susan Boyle.

Alternatively, your girlfriend may be Susan Jameson, so you might feel it 
appropriate for "sms:love:Hello there" to go to Susan Jameson.

In short, I'm proposing the addition of aliases for contacts. 
```

Original issue reported on code.google.com by `Eckankar` on 20 Feb 2011 at 10:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 21 Feb 2011 at 11:03
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I have begun working on a general DB Backend Framework for GtalkSMS. There are 
many other information that should be stored persistent in a database, but the 
alias information will be the first.
```

Original comment by `fschm...@gmail.com` on 7 Mar 2011 at 4:14
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 7 Mar 2011 at 7:16
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Changeset 4a84348ff4 - 
http://code.google.com/p/gtalksms/source/detail?r=4a84348ff4edb0bc6f40bf8a25133f
1a4b6c3975
```

Original comment by `fschm...@gmail.com` on 10 Mar 2011 at 9:46
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.1
```

Original comment by `Florent....@gmail.com` on 14 Mar 2011 at 10:42
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
